nx/test/nx_test.exs

```diff
@@ -1628,4 +1628,51 @@ defmodule NxTest do
                )
     end
   end
+
+  if Version.match?(System.version(), ">= 1.13.0-dev") do
```

Comment:
Move this conditional to only wrap around the tests that use eval and therefore requires the modifiers.